lib/command/apply_template.rb

```diff
@@ -126,7 +127,7 @@ def confirm_workload(template)
     def apply_template(filename)
       data = File.read(filename)
                  .gsub("APP_GVC", config.app)
-                 .gsub("APP_LOCATION", config[:default_location])
+                 .gsub("APP_LOCATION", config.options[:location] || config[:default_location])
```

Comment:
We could do it like `org_option` and `app_option`, and add a `default` key with the value `config[:default_location]`. See:

```
def self.org_option(required: false) # rubocop:disable Metrics/MethodLength
  {
    name: :org,
    params: {
      aliases: ["-o"],
      banner: "ORG_NAME",
      desc: "Organization name",
      type: :string,
      required: required,
      default: ENV.fetch("CPLN_ORG", nil) # <----------
    }
  }
end
```

Reply:
It seems at this stage, we still don't have `config`.

Reply:
Hm, you're right, this is a class method, so it doesn't get initialized with `config`.

In that case, we can do this when initializing the `Config` class (after loading the app config and apps). Something like:

```
@location = options[:location] || config[:default_location]
```

And then have `location` in `attr_reader` so we can access it like `config.location` elsewhere.

Reply:
I thought about it earlier. I wasn't sure about it because I am not sure of the criteria for adding methods to the `config`.

If you think this is an appropriate thing to do, I can add it.

Reply:
`config` is there though, you just have to do it after `load_app_config`, which is what loads `config`.